src/platform/silabs/wifi/wiseconnect-interface/WiseconnectWifiInterface.cpp

```diff
@@ -176,8 +145,12 @@ void ResetDHCPNotificationFlags(void)
 
     ResetIPNotificationStates();
     hasNotifiedWifiConnectivity = false;
-
+#if SLI_SI917
+    // Setting the notify since 917 device got the IP already
+    WifiPlatformEvent event = WifiPlatformEvent::kStationNotify;
```

Comment:
the function name is not indicative of the event being posted here.

Reply:
I have updated the names, I can even add these 2 in different files if needed